__tests__/app.test.js

```diff
@@ -491,20 +491,22 @@ describe("/api/comments/:comment_id", () => {
         .delete("/api/comments/1")
         .expect(204)
         .then(() => {
-          return db.query(`SELECT * FROM comments`).then(({ rows }) => {
-            const allCommentIds = rows.map((row) => {
-              return row.comment_id;
+          return db
+            .query(`SELECT * FROM comments WHERE comment_id = 1`)
+            .then(({ rows }) => {
+              const allCommentIds = rows.map((row) => {
```

Comment:
Just a small aside, but i think `rows` would come back as an empty array, so you're mapping through and empty array. 

it works though, but may not be needed.